### PR TITLE
Add e2e tests on dashboard features

### PR DESCRIPTION
**Pull request by saw-jan**

### CODE REVIEW

File: frontend/tests/acceptance/features/webUIDashboard/dashboard.feature [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+  Background: User has access to dashboard
+    Given a user has been registered with the following details:
+      | name | email          | password |
+      | test | test@email.com | testpass |
+    And the user has logged in to the dashboard with email "test@email.com" and password "testpass" using the webUI
+
+  @movejobs
+  Scenario Outline: Move job application from one status heading to another status heading
+		Given the following job application already exists:
```
Comment (kiranparajuli589) [inline frontend/tests/acceptance/features/webUIDashboard/dashboard.feature]: check indentation!

File: frontend/tests/acceptance/features/webUIDashboard/dashboard.feature [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      | title   | company   | priority   | status          |
+      | <title> | <company> | <priority> | <initialStatus> |
+    When the user moves job application from "<initialStatus>" to "<dragTOStatus>" status board
```
Comment (kiranparajuli589) [inline frontend/tests/acceptance/features/webUIDashboard/dashboard.feature]:
```suggestion
    When the user moves job application from "<initialStatus>" to "<dragToStatus>" status board using the webUI
```

File: frontend/tests/acceptance/features/webUIDashboard/dashboard.feature [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      | <title> | <company> | <priority> | <status> |
+    When the user deletes job application of "<status>" status board using webUI
+    Then the job application of title "<title>" should not appear in "<status>" status board
+    Examples:
+      | title          | company     | priority | status       |
+      | Office Manager | ABC Company | Medium   | Yet to Apply |
+      | Office Manager | ABC Company | Medium   | Applied      |
+      | Office Manager | ABC Company | Medium   | In Progress  |
+      | Office Manager | ABC Company | Medium   | Accepted     |
+      | Office Manager | ABC Company | Medium   | Rejected     |
```
Comment (Talank) [inline frontend/tests/acceptance/features/webUIDashboard/dashboard.feature]: `title`, `Company`, and `priority` have a single value in all rows. You do not need to keep it in the Examples table.

File: frontend/tests/acceptance/features/webUIDashboard/dashboard.feature [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    Then the user should be redirected to update job application page and should see following data:
+      | title   | company   | priority   | status   |
+      | <title> | <company> | <priority> | <status> |
+    Examples:
+      | title          | company     | priority | status       |
+      | Office Manager | ABC Company | Medium   | Yet to Apply |
+      | Office Manager | ABC Company | Medium   | Applied      |
+      | Office Manager | ABC Company | Medium   | In Progress  |
+      | Office Manager | ABC Company | Medium   | Accepted     |
+      | Office Manager | ABC Company | Medium   | Rejected     |
```
Comment (Talank) [inline frontend/tests/acceptance/features/webUIDashboard/dashboard.feature]: same as above.
`title`, `Company`, and `priority` have a single value in all rows. You do not need to keep it in the Examples table.

File: frontend/tests/acceptance/step_definitions/dashboardContext.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+);
+
+Then(
+  'the second job applications should appear above the first job application within {string} status board',
+  async (status) => {
+    within(dashboard.getJobStatusBoard(status), () => {
+      I.see(secondJob, dashboard.getCardPosition(status, 1));
+      I.see(firstJob, dashboard.getCardPosition(status, 2));
+    });
+    await tearDown();
```
Comment (kiranparajuli589) [inline frontend/tests/acceptance/step_definitions/dashboardContext.js:78]: :bow: haripp :smile:

**Review by kiranparajuli589 — CHANGES_REQUESTED** [review]
Try to use less hardcoded values in your code! Otherwise LGTM :+1:

File: frontend/tests/acceptance/step_definitions/dashboardContext.js [hunk 6]
```diff
@@ -0,0 +1,114 @@
+const { I } = inject();
+const dashboard = require('../pages/dashboardPage');
+const { addJobApplication, deleteJobApplication } = require('../helpers/api/job_application');
+const USER_API = require('../helpers/api/user');
+const updateJob = require('../pages/updateJobApplicationPage');
+
+let Token, firstJob, secondJob;
```
Comment (Talank) [inline frontend/tests/acceptance/step_definitions/dashboardContext.js]: IMO, it is better variable name should start from lower case letter to follow the convention on this project. Rename the variable `Token` .

**Pull request closed without merging**